src/tables/ctran_data.py

```diff
@@ -0,0 +1,142 @@
+import sys
+import pandas
+from .table import Table
+from sqlalchemy.exc import SQLAlchemyError
+
+class CTran_Data(Table):
+
+    ###########################################################################
+    # Public Methods
+
+    def __init__(self, user=None, passwd=None, hostname="localhost", db_name="aperature", verbose=False, engine=None):
+        super().__init__(user, passwd, hostname, db_name, verbose, engine)
+        self._schema = "aperature"
+        self._table_name = "ctran_data"
+        self._index_col = "data_row"
+        self._expected_cols = set([
+            "service_date",
+            "vehicle_number",
+            "leave_time",
+            "train",
+            "badge",
```

Comment:
"badge" must be deleted. It must be removed from all of our sample data immediately. ~Kristin